### PR TITLE
Add Getting Started

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Add getting started C#

https://deploy-preview-34--doc-sdk-csharp.netlify.com/getting-started/standalone/

### CODE REVIEW

File: doc/1/getting-started/standalone/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+description: Getting started with Kuzzle and C#
+order: 0
+---
+
+# Getting Started with Kuzzle and C#
+
+This tutorial explains you how to use **Kuzzle** with **C#**, **.NET Core SDK 2.1** and the **Kuzzle C# SDK**.
+It will walk you through creating scripts that can **store** documents in Kuzzle and subscribe to **notifications** about document creations.
+
+You are going to write an application that **stores** documents in Kuzzle Server and subscribe to **real time notifications** for each created document.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
You are going to write an application that **stores** documents in Kuzzle Server and subscribes to **real time notifications** for each created document.
```

File: doc/1/getting-started/standalone/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+To follow this tutorial, you must have a Kuzzle Server up and running. Follow these instructions if this is not already the case: [Running Kuzzle](/core/1/guides/getting-started/running-kuzzle/).
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md:16]:
```suggestion
To follow this tutorial, you must have a Kuzzle Server up and running. Follow these steps if it's not done yet: [Running Kuzzle](/core/1/guides/getting-started/running-kuzzle/).
```

File: doc/1/getting-started/standalone/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+
+::: info
+Having trouble? Get in touch with us on [Gitter](https://gitter.im/kuzzleio/kuzzle)!
+:::
+
+## Explore the SDK
+
+It's time to get started with the [Kuzzle C# SDK](/sdk/csharp/1). This section, explains you how to store a document and subscribe to notifications in Kuzzle using the C# SDK.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
It's time to start with the [Kuzzle C# SDK](/sdk/csharp/1). This section explains you how to store a document and subscribe to notifications in Kuzzle using the C# SDK.
```

File: doc/1/getting-started/standalone/index.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+Before proceeding, please make sure your system has **.NET Core SDK** version 2.1 or higher.
+
+::: warning
+If you're using monodevelop on Linux, you'll need at least mono 5.20+ (w/ msbuild 16+). Due to compatibility problems, you HAVE TO install .NET Core SDK 2.1, if you only have the 2.2 one, you won't be able to build the project with msbuild (which monodevelop uses).
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
If you're using monodevelop on Linux, you'll need at least mono 5.20+ (w/ msbuild 16+). Due to compatibility issues, you HAVE TO install .NET Core SDK 2.1, if you only have the 2.2 one, you won't be able to build the project with msbuild (which monodevelop uses).
```

File: doc/1/getting-started/standalone/index.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+Create your playground directory, initialize a new console application using dotnet CLI and add reference to the [Kuzzle SDK package](https://www.nuget.org/packages/kuzzlesdk/).  
+
+```sh
+mkdir "kuzzle-playground"
+cd "kuzzle-playground"
+dotnet new console
+dotnet add package kuzzlesdk
+```
+
+Then inside the `Program.cs` file, we will create a console application that take an argument to either initialize the application, subscribe to notification or create a document.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Then inside the `Program.cs` file, we will create a console application that takes an argument to either initialize the application, subscribe to notification or create a document.
```

File: doc/1/getting-started/standalone/index.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+<<< ./snippets/Switch.cs
+
+Then create a `GetSdk` function to instantiate the SDK and connects it to a Kuzzle instance using the WebSocket protocol.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Then create a `GetSdk` function to instantiate the SDK and connect it to a Kuzzle instance using the WebSocket protocol.
```

File: doc/1/getting-started/standalone/index.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+After, connect the client to your Kuzzle server with the [Kuzzle.ConnectAsync](/sdk/csharp/1/core-classes/kuzzle/connect-async) method.
+
+<<< ./snippets/Program1.cs:1
+
+:::info
+Replace 'kuzzle' which is the Kuzzle server hostname with 'localhost' or with the host name where your Kuzzle server is running.
+:::
+
+Afterwards you have to add the code that will access Kuzzle to create a new index 'nyc-open-data' and a new collection 'yellow-taxi' that you will use to store data later on.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Afterwards you have to add the code that will access Kuzzle to create a new index 'nyc-open-data' and a new collection 'yellow-taxi' that you will use to store data.
```

File: doc/1/getting-started/standalone/index.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+
+<<< ./snippets/Program.cs:4
+
+Run the code with dotnet:
+
+```bash
+dotnet run -- subscribe
+```
+
+The  program is now running endlessly, waiting for notifications about documents matching its filters, specifically documents that have a `license` field equal to `'B'`.
```
Comment (y-abs) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
The program is now running endlessly, waiting for notifications about documents matching its filters, specifically documents that have a `license` field equal to `'B'`.
```

**Review by y-abs — CHANGES_REQUESTED** [review]

File: doc/1/getting-started/standalone/index.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+code: false
+type: page
+title: C# Standalone
+description: Getting started with Kuzzle and C#
+order: 0
+---
+
+# Getting Started with Kuzzle and C#
+
+This tutorial explains you how to use **Kuzzle** with **C#**, **.NET Core SDK 2.1** and the **Kuzzle C# SDK**.
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
This tutorial explains how to use **Kuzzle** with **C#**, **.NET Core SDK 2.1** and the **Kuzzle C# SDK**.
```

File: doc/1/getting-started/standalone/index.md (hunk 2 above)
Comment (scottinet) [reply doc/1/getting-started/standalone/index.md:16]: I find the original version better than this proposal.

File: doc/1/getting-started/standalone/index.md (hunk 3 above)
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
It's time to get started with the [Kuzzle C# SDK](/sdk/csharp/1). This section explains how to store a document and how to subscribe to notifications in Kuzzle using the C# SDK.
```

File: doc/1/getting-started/standalone/index.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+Before proceeding, please make sure your system has **.NET Core SDK** version 2.1 or higher.
+
+::: warning
+If you're using monodevelop on Linux, you'll need at least mono 5.20+ (w/ msbuild 16+). Due to compatibility problems, you HAVE TO install .NET Core SDK 2.1, if you only have the 2.2 one, you won't be able to build the project with msbuild (which monodevelop uses).
+:::
+
+## Prepare your environment
+
+Create your playground directory, initialize a new console application using dotnet CLI and add reference to the [Kuzzle SDK package](https://www.nuget.org/packages/kuzzlesdk/).
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Create your playground directory, initialize a new console application using dotnet CLI and add a reference to the [Kuzzle SDK package](https://www.nuget.org/packages/kuzzlesdk/).  
```

File: doc/1/getting-started/standalone/index.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+dotnet add package kuzzlesdk
+```
+
+Then inside the `Program.cs` file, we will create a console application that take an argument to either initialize the application, subscribe to notification or create a document.
+
+<<< ./snippets/Switch.cs
+
+Then create a `GetSdk` function to instantiate the SDK and connects it to a Kuzzle instance using the WebSocket protocol.
+
+After, connect the client to your Kuzzle server with the [Kuzzle.ConnectAsync](/sdk/csharp/1/core-classes/kuzzle/connect-async) method.
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Once done, connect the client to your Kuzzle server using the [Kuzzle.ConnectAsync](/sdk/csharp/1/core-classes/kuzzle/connect-async) method.
```

File: doc/1/getting-started/standalone/index.md (hunk 7 above)
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
Now, you need to create a new 'nyc-open-data'  index, holding a new 'yellow-taxi' collection. This structure will be used later on to store data.
```

File: doc/1/getting-started/standalone/index.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+Your `Program.cs` file should now look like this:
+
+<<< ./snippets/Program1.cs
+
+This code does the following:
+
+- creates an instance of the SDK
+- connects it to Kuzzle running on `kuzzle` (change the hostname if needed) using WebSocket
+- creates the `nyc-open-data` index
+- creates the `yellow-taxi` collection (within the `nyc-open-data` index)
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
- creates the `yellow-taxi` collection within the `nyc-open-data` index
```

File: doc/1/getting-started/standalone/index.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+
+Run the code with dotnet:
+
+```bash
+dotnet run -- subscribe
+```
+
+The  program is now running endlessly, waiting for notifications about documents matching its filters, specifically documents that have a `license` field equal to `'B'`.
+
+We will add a `await Task.Delay(10000, token.Token);` after a successfull subscribe to keep the program running. The `CancellationToken` will allow us to stop the thread after receiving a notification.
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]:
```suggestion
We added a `await Task.Delay(10000, token.Token);` line after a successfull subscribe to keep the program running until after a notification has been received.
```

File: doc/1/getting-started/standalone/index.md [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+
+Now in another terminal, launch the program to create a document:
+
+```bash
+dotnet run -- create
+```
+
+This creates a new document in Kuzzle which, in turn, triggers a [document notification](/core/1/api/essentials/notifications/#documents-changes-messages) sent to the program who subscribes.
```
Comment (scottinet) [inline doc/1/getting-started/standalone/index.md]: Hard to understand, I hope that the following proposal is to the point:

```suggestion
This creates a new document in Kuzzle which, in turn, triggers a [document notification](/core/1/api/essentials/notifications/#documents-changes-messages). That notification is sent by Kuzzle to our other running program, which subscribed to changes occuring in that index and collection.
```

**Review by scottinet — CHANGES_REQUESTED** [review]

File: doc/1/getting-started/standalone/index.md (hunk 3 above)
Comment (Aschen) [reply doc/1/getting-started/standalone/index.md]: Sorry, I pick Seb :trollface:

**Review by scottinet — APPROVED** [review]

**Review by y-abs — APPROVED** [review]

**Pull request merged**